modules/monitoring-configuring-a-persistent-volume-claim.adoc

```diff
@@ -125,58 +102,21 @@ metadata:
   namespace: openshift-user-workload-monitoring
 data:
   config.yaml: |
-    <component>:
+    <component>: #<1>
       volumeClaimTemplate:
         spec:
-          storageClassName: <storage_class>
+          storageClassName: <storage_class> #<2>
           resources:
             requests:
-              storage: <amount_of_storage>
+              storage: <amount_of_storage> #<3>
 ----
+<1> Specify the component for user-defined monitoring for which you want to configure the PVC.
+<2> Specify an existing storage class. It you do not specify any, the default storage class is used.
```

Comment:
```suggestion
<2> Specify an existing storage class. If you do not specify any, the default storage class is used.
```

Reply:
oh thank you for noticing! I am all over the place this morning, so I truly value your reviews +1